src/app/services/utility.service.ts

```diff
@@ -0,0 +1,8 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UtilityService {
+  showSplash = true;
```

Comment:
service should not be responsible for maintaining a state.

Reply:
Vaia, can you please suggest how should I manage state without using service?